scripts/capture.py

```diff
@@ -0,0 +1,45 @@
+import cv2
+import datetime
+import os
+import sys
+import optparse
+
+OUTPUT_DIR = '/home/pi/data'
+DIR = '%Y-%m-%d'
+FILE = '%H:%M:%S.png'
+
+
+def capture_and_save(warmup=15):
+    cap = cv2.VideoCapture(0)
+
+    #  warmup camera
+    for i in range(15):
```

Comment:
@Adman there is `warmup` now, it seems that it should be used rather than a hardcoded `15`.